pkgs/development/python-modules/tinygrad/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "tinygrad";
+  version = "0.7.0";
+  pyproject = true;
```

```suggestion
  pyproject = true;
  
  disabled = pythonOlder "3.8";
```
https://github.com/tinygrad/tinygrad/blob/8b354b3f73116e3acaad41cf55281de242709506/setup.py#L23